apps/system/js/remote/message_controller.js

```diff
@@ -27,7 +33,7 @@
     },
 
     _start: function() {
-      this.displayId = Service.query('displayId');
+      this.displayId = parseInt(Service.query('displayId'));
```

Comment:
By design, `displayId` should always return an integer so maybe you don't need to do `parseInt` here.